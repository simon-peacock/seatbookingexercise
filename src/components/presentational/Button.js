import React from 'react'

import { Button } from 'semantic-ui-react'

export default (props) => {

  return (
    <Button disabled={props.disabled}>{props.title}</Button>
  )
}
