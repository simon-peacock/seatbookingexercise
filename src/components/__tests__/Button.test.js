import React from 'react';
import Button from 'Presentational/Button'
import renderer from 'react-test-renderer';

test('Button renders correctly', () => {
  const component = renderer.create(<Button
    title={'Next step'}
    disabled={false}
  />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
