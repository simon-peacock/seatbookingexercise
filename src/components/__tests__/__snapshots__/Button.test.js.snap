// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  role="button"
>
  Next step
</button>
`;
